sailfishusermanager.cpp

```diff
@@ -211,6 +211,22 @@ uint SailfishUserManager::addUser(const QString &name)
         return 0;
     }
 
+    int count = 0;
+    struct group *grent = getgrnam(USER_GROUP);
+    if (grent) {
+        for (int i = 0; grent->gr_mem[i]; i++) {
+            if (getpwnam(grent->gr_mem[i]))
+                count++;
+        }
+    }
+    if (count > SAILFISH_USERMANAGER_MAX_USERS) {
+        // Just > above because master user is also member of users group
```

Comment:
AFAIK The number of users we can have is 7 and that includes the master user. So, already having 7 users and still proceeding to add 8th should be wrong - or am I missing something?

Reply:
Luks has eight slots where one is reserved for master or are there more reservations?

Reply:
One of them is reserved for backup. So 7 users including master...

Reply:
Ok, fixing this in the pc version fix commit.

Reply:
Sorry to flog a dead horse / already merged pr, but AFAICT it should be "if (count >= 7)" not "if (count > 7)". We have 8 luks slots, 7 of those available for 7 users - one of which happens to be the master. Which yields SAILFISH_USERMANAGER_MAX_USERS = 7 -> counting already present users, finding out that we have 7 of them, and still proceeding to create 8th one = wrong / I am missing something fundamental here.

Reply:
oh well, it was fixed in not-closed pr ;-)